src/test/java/automation/app/Bookingdates.java

```diff
@@ -0,0 +1,18 @@
+package automation.app;
+
+import java.time.LocalDate;
+import java.util.Date;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class Bookingdates {
+
+	private String checkin;
+	private String checkout;
```

Comment:
Instead of using strings you can use this: https://jenkov.com/tutorials/java-internationalization/simpledateformat.html